scilifelab/lims_utils/workset_upload.py

```diff
@@ -88,6 +89,13 @@ def __init__(self, lims, crawler, log):
         self.obj['technician']=crawler.starting_proc.technician.initials
         self.obj['id']=crawler.starting_proc.id
         self.obj['date_run']=crawler.starting_proc.date_run
+        self.obj['workset_notes']={}
+        if "Workset Notes" in crawler.starting_proc.udf:
+            notes=json.loads(crawler.starting_proc.udf["Workset Notes"])
+            for date, note in notes:
```

Comment:
ha, it's an artefact of the time where I wanted to store the notes in statusdb. sorry